privacyidea/cli/privacyideatokenjanitor/findtokens.py

```diff
@@ -452,6 +453,8 @@ def export_user_data(token_list, attributes=None):
 @click.option('--active', help='True|False|None')
 @click.option('--orphaned',
               help='Whether the token is an orphaned token. Set to 1')
+@click.option('--default_for_orphaned_exception', default=True, type=bool,
```

Comment:
I called it `--orphaned-on-error` in the `pi-tokenjanitor`, maybe we should use the same option name.